### PR TITLE
"Whats new" New Azure and GCP native log forwarding (Publish date 03/03/22)

### PR DESCRIPTION
**Pull request by ymirelesNewRelic**
All existing New Relic customers using Azure or GCP cloud infrastructure integrations can now send logs to New Relic without the need for unsupported forwarders or custom solutions published by customer success teams.

### CODE REVIEW

File: src/content/whats-new/2022/02/whats-new-2-28-azure-gcp-logs-campaign.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+New logs integrations:
+
+* **For Azure**, Blob Storage, or via an Event Hub.
+* **For GCP**, pub/sub integration.
+
+!["Azure and CGP Integration Icons"](./images/azure_gcp_logs_icons.png "Azure and CGP Integration Icons")
+
+<figcaption>Azure and GCP log integrations are part of New Relic’s log forwarding plugins that make it easy for our customers to get logs into New Relic One.</figcaption>
+
+To start using logs with Azure or GCP, or any of New Relic’s native log integrations simply type *‘logs’* by clicking on [**+ Add More Data**](https://one.newrelic.com/nr1-core?account=[]&state=7db6a5b5-d270-7c21-d96e-af62e2a6c95f) on your NR1 homepage.
```
Comment (alexiskjones) [inline src/content/whats-new/2022/02/whats-new-2-28-azure-gcp-logs-campaign.md:21]: New Relic One should be spelled out (not NR1)

Comment (ymirelesNewRelic) [reply src/content/whats-new/2022/02/whats-new-2-28-azure-gcp-logs-campaign.md:21]: Noted, will be corrected once the PR goes live.

**Review by ymirelesNewRelic — COMMENTED** [review]
this will be changed once the post goes live

**Comment by barbnewrelic** [comment]
@ymirelesNewRelic  The filename says Feb 28 but the release date in the doc itself says March 3. Can you please confirm when this is expected to go live? Thanks!

**Comment by ymirelesNewRelic** [comment]
Please go live on March 3rd. 2022

Thank you
--
Yoram Mireles
Principal Product Marketing Manager
***@***.***

File: src/content/whats-new/2022/02/whats-new-2-28-azure-gcp-logs-campaign.md (hunk 1 above)
Comment (rhetoric101) [reply src/content/whats-new/2022/02/whats-new-2-28-azure-gcp-logs-campaign.md]: @alexiskjones, I revised these instructions slightly by just asking users to click the link, which seems to handle everything.

```
To start using logs with Azure or GCP, or any of New Relic’s native log integrations:

1. Click this [**+ Add More Data**](https://one.newrelic.com/nr1-core?account=[]&state=7db6a5b5-d270-7c21-d96e-af62e2a6c95f) link.
2. In the search bar, type **logs**, and press Enter to see your options.

```

**Comment by rhetoric101** [comment]
Note to docs hero: I completed a basic edit. Please double-check before publishing on March 3rd.

**Review by bradleycamacho — APPROVED** [review]

**Comment by bradleycamacho** [comment]
@alexiskjones, can you confirm this is approved by the PMMs?

**Comment by ymirelesNewRelic** [comment]
I am the PMM, and I approve.

Thank you
--
*Yoram Mireles*
Principal Product Marketing Manager
***@***.***

**Comment by bradleycamacho** [comment]
@alexiskjones Ah my bad! Merging in now, thanks for your patience!

**Comment by ymirelesNewRelic** [comment]
Thank you!

**Pull request merged**